tests/stability/test_deadlock.py

```diff
@@ -2,11 +2,9 @@
 
 import coiled.v2
 import dask
-import pytest
 from distributed import Client, wait
 
 
-@pytest.mark.skip(reason="https://github.com/dask/distributed/issues/6110")
 def test_repeated_merge_spill():
```

Comment:
Can you mark it to run only on the latest version of dask? The tests are going to fail for the coiled runtime version 0.0.3 which has a hard pin. 
I think you can decorate it with `@pytest.mark.latest_runtime` and that should work.

Reply:
Fine with me. FWIW, for these things I think we should have a `skipif(version<version_number)` such that we run the tests for all versions above this, not just the latest